modules/evs/fsw/inc/cfe_evs_msg.h

```diff
@@ -0,0 +1,1287 @@
+/*
+**  GSC-18128-1, "Core Flight Executive Version 6.7"
+**
+**  Copyright (c) 2006-2019 United States Government as represented by
+**  the Administrator of the National Aeronautics and Space Administration.
+**  All Rights Reserved.
+**
+**  Licensed under the Apache License, Version 2.0 (the "License");
+**  you may not use this file except in compliance with the License.
+**  You may obtain a copy of the License at
+**
+**    http://www.apache.org/licenses/LICENSE-2.0
+**
+**  Unless required by applicable law or agreed to in writing, software
+**  distributed under the License is distributed on an "AS IS" BASIS,
+**  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+**  See the License for the specific language governing permissions and
+**  limitations under the License.
+*/
+
+/**
+ * @file
+ *
+ *  Title:    Event Services Message definition header file Header File
+ *
+ *  Purpose:
+ *	           Unit specification for Event services command codes and data structures.
+ *
+ *  Design Notes:
+ *
+ *  References:
+ *     Flight Software Branch C Coding Standard Version 1.0a
+ *
+ */
+
+#ifndef CFE_EVS_MSG_H
+#define CFE_EVS_MSG_H
+
+/********************************** Include Files  ************************************/
+#include "common_types.h"            /* Basic data types */
+#include "cfe_msg_hdr.h"             /* for header definitions */
+#include "cfe_evs_extern_typedefs.h" /* for EVS-specific types such as CFE_EVS_LogMode_Enum_t */
+#include "cfe_es_extern_typedefs.h"  /* for CFE_ES_AppId_t type */
+
+/** \name Event Services Command Codes */
+/** \{ */
+
+/** \cfeevscmd Event Services No-Op
+**
+**  \par Description
+**       This command performs no other function than to increment the
+**       command execution counter. The command may be used to verify
+**       general aliveness of the Event Services task.
+**
+**  \cfecmdmnemonic \EVS_NOOP
+**
+**  \par Command Structure
+**       #CFE_EVS_NoopCmd_t
+**
+**  \par Command Verification
+**       Successful execution of this command may be verified with the
+**       following telemetry:
+**       - \b \c \EVS_CMDPC - command execution counter will
+**         increment
+**       - The #CFE_EVS_NOOP_EID informational event message will
+**         be generated
+**
+**  \par Error Conditions
+**       There are no error conditions for this command. If the Event
+**       Services receives the command, the event is sent (although it
+**       may be filtered by EVS itself) and the counter is incremented
+**       unconditionally.
+**
+**  \par Criticality
+**       None
+**
+**  \sa
+*/
+#define CFE_EVS_NOOP_CC 0
```

Comment:
all of these look like they are an enum

Reply:
Similar concepts (will open issue for const variables).